convert-apps.sh

```
@@ -579,13 +579,17 @@ get_existing_runtipi_port() {
+    local port_map_file="$OUTPUT_DIR/.port_map"
     
-    # Clear any existing port tracker
+    # Clear any existing port tracker files
     > "$port_track_file"
+    > "$port_map_file"
     
     # For each platform, scan existing apps and load their ports
     for platform in "${PLATFORMS[@]}"; do
-        local workspace_dir="$(dirname "$OUTPUT_DIR")"
+        # Get the workspace directory (parent of the script directory)
+        local script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)